hotline/notifications/templatetags/notifications.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+from hotline.notifications.models import UserNotificationQuery
+
+register = template.Library()
+
+
+@register.filter
+def notification_url(entry):
```

Comment:
I don't *think* you need a template tag to do this. When you're looping through the form, I'm pretty sure there is an easy way to get access to the underlying model object associated with the choice.